### PR TITLE
Implement puppetdb exporter

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by sebastianrakel — COMMENTED** [review]
Why is there memcached stuff?

File: REFERENCE.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ _Public Classes_
 Currently only the install from url is implemented, when Prometheus will deliver packages for some Linux distros I will
 implement the package install method as well
 The package method needs specific yum or apt repo settings which are not made yet by the module
+* [`prometheus::memcached_exporter`](#prometheusmemcached_exporter): This module manages prometheus node memcached_exporter
```
Comment (sebastianrakel) [inline REFERENCE.md:26]: Why is there memcached stuff

Comment (bastelfreak) [reply REFERENCE.md:26]: the reference.md is generated for all classes in this module. during changes on the memchaced_exporter class, the file wasn't regenerated.

Comment (sebastianrakel) [reply REFERENCE.md:26]: okay. can't be changed? delete the line and extra PR for this line?

**Review by sebastianrakel — APPROVED** [review]

**Pull request merged**